container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1292,6 +1305,119 @@ func (manager *DockerManager) FetchImage(ctx context.Context, image string, down
 	return pulledFromRemote, imageArchitecture, nil
 }
 
+func (manager *DockerManager) BuildImage(ctx context.Context, imageName string, imageBuildSpec *image_build_spec.ImageBuildSpec) (string, error) {
+	contextDirPath := imageBuildSpec.GetContextDirPath()
+	containerImageFileTarReader, err := getBuildContextReader(contextDirPath)
```

Comment:
is containerImage or buildContextTar ? because the var name is `containerImageFileTarReader`

Reply:
I think the var name can be something like `containerBuildCtxTarReader`

Reply:
ahh good shout